package.json

```diff
@@ -36,7 +36,7 @@
     "test:debug": "mocha debug test/runner.js --reporter spec"
   },
   "dependencies": {
-    "broccoli-caching-writer": "^0.5.4",
+    "broccoli-caching-writer": "^1.1.0",
```

Comment:
i think i mitigate the largest chunk of perf issues: https://github.com/gabrielgrant/broccoli-less-single/pull/33